execution_mode/grpc_utils/mario.proto

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
after implementing this proto file and run `build.sh`, the two `mario_pb2*` files will be automatically generated